vmm/src/device_manager.rs

```diff
@@ -4127,6 +4129,26 @@ impl Pausable for DeviceManager {
                 migratable.lock().unwrap().pause()?;
             }
         }
+        // On AArch64, the pause of device manager will trigger a "pause" of GIC,
```

Comment:
Nit: Do you mean the following with the comment?

" On AArch64, the "pause" of the device manager `needs to` trigger a "pause" of GIC, ... "

Reply:
Done.